images/Dockerfile-clientserver

```diff
@@ -1,12 +1,34 @@
-FROM quay.io/redhat-user-workloads/rhtas-tenant/rhtas-cli-1-0-beta/cosign@sha256:1a34ac0ee18d144b18a3f18333216e87709018acb6fea42892b5e1a17e2c8af7 AS cosign-image
-FROM quay.io/redhat-user-workloads/rhtas-tenant/rhtas-cli-1-0-beta/gitsign@sha256:a8e8fb362beb5a9a10f11877f676d4d6399b4f66fe9bd15c3bc352748707b98d AS gitsign-image
+FROM quay.io/redhat-user-workloads/rhtas-tenant/rhtas-cli-1-0-beta/cosign@sha256:82812a94eb64f6359c5a3adc43be03844b54a6ebe9c025cf010e88067c1ac5f3 AS cosign-image
+FROM quay.io/redhat-user-workloads/rhtas-tenant/rhtas-cli-1-0-beta/gitsign@sha256:f89115971b1ede4cfa5874f90c90c8c2d79de534fd84094b1a9e626c80394f1b AS gitsign-image
+FROM quay.io/redhat-user-workloads/rhtas-tenant/rhtas-cli-1-0-beta/rekor-cli@sha256:7da0656cc35c0fce3e62f80d8a1e72eac78a4caf6e636b27e598b98b77d8f9b3 AS rekor-image
 
 FROM registry.redhat.io/rhel8/httpd-24:latest
 
 RUN mkdir -p /var/www/html/clients
+RUN mkdir -p /var/www/html/clients/darwin
+RUN mkdir -p /var/www/html/clients/linux
+RUN mkdir -p /var/www/html/clients/windows
 
-COPY --from=cosign-image /usr/local/bin/cosign /var/www/html/clients/
-COPY --from=gitsign-image /usr/local/bin/gitsign /var/www/html/clients/
+
+COPY --from=cosign-image /usr/local/bin/cosign-linux-amd64.gz /var/www/html/clients/linux
```

Comment:
change these to 
```
COPY --from=cosign-image /usr/local/bin/cosign-linux-amd64.gz /var/www/html/clients/linux/cosign
```
for all - ie remove the -linux-amd64/etc... from the final binary name

Reply:
Will i still be able to uncompress these if the extension isnt there?

Reply:
probably not, but I see you've worked around it :)